modules/platform/nuxeo-platform-audit/nuxeo-platform-audit-core/src/test/java/org/nuxeo/ecm/platform/audit/io/LogEntryJsonWriterTest.java

```diff
@@ -114,4 +119,42 @@ public void testArrayInExtendedInfo() throws Exception {
         json.has("extended").properties(1).has("params").isArray().contains("a simple string");
     }
 
+    @Test
+    public void testMapInExtendedInfo() throws Exception {
+        Map<String, ExtendedInfo> infos = new HashMap<>();
+
+        HashMap<String, Serializable> infoMap = new HashMap<>();
+        infoMap.put("String","abcde");
+        Date now = new Date();
+        infoMap.put("Date", now);
+        infoMap.put("Boolean",false);
+        infoMap.put("Integer",1);
+        infoMap.put("Double",2.0);
+        infos.put("params", ExtendedInfoImpl.createExtendedInfo(infoMap));
+
+        LogEntry logEntry = new LogEntryImpl();
+        logEntry.setExtendedInfos(infos);
+
+        JsonAssert json = jsonAssert(logEntry);
+        json.properties(14);
+        json.has("entity-type").isEquals("logEntry");
+        json.has("id").isEquals(0);
+        json.has("category").isNull();
+        json.has("principalName").isNull();
+        json.has("docPath").isNull();
+        json.has("docType").isNull();
+        json.has("docUUID").isNull();
+        json.has("eventId").isNull();
+        json.has("repositoryId").isNull();
+        json.has("eventDate").isText();
+        json.has("logDate").isText();
+        json.has("comment").isNull();
+        json.has("docLifeCycle").isNull();
+        json.has("extended").properties(1).has("params").isObject().has("String").isEquals("abcde");
+        json.has("extended").properties(1).has("params").isObject().has("Date").isEquals(formatISODateTime(now));
+        json.has("extended").properties(1).has("params").isObject().has("Integer").isEquals(1);
+        json.has("extended").properties(1).has("params").isObject().has("Double").isEquals(2.0,0.0);
+        json.has("extended").properties(1).has("params").isObject().has("Boolean").isEquals(false);
```

Comment:
```suggestion
        var extended = json.has("extended").properties(1).has("params").isObject();
        extended.has("String").isEquals("abcde");
        extended.has("Date").isEquals(formatISODateTime(now));
        extended.has("Integer").isEquals(1);
        extended.has("Double").isEquals(2.0,0.0);
        extended.has("Boolean").isEquals(false);
```

Reply:
Yes but avoid using `var` if you want an easy backport

Reply:
done without var